src/transport/SessionManager.cpp

```diff
@@ -387,6 +387,20 @@ void SessionManager::ExpireAllPASEPairings()
     });
 }
 
+void SessionManager::ReleaseSessionsForFabricExceptOne(FabricIndex fabricIndex, const SessionHandle & deferred)
```

Comment:
I think the exchange manager should be responsible for calling this function when `AbortExchangesForFabricExceptOne` is called, given the tight coupling between here and there in terms of the exchange lifetime management.